src/utils/getBoundsForNode.ts

```diff
@@ -38,14 +38,19 @@ export function getBoundsForNode(
   const { scrollTop, scrollLeft } = containerScroll
   const { documentScrollTop, documentScrollLeft } = getDocumentScroll()
 
-  const rect = node.getBoundingClientRect()
-
-  return {
-    top: rect.top + documentScrollTop + scrollTop,
-    left: rect.left + documentScrollLeft + scrollLeft,
-    offsetWidth: node.offsetWidth,
-    offsetHeight: node.offsetHeight,
-    width: rect.width,
-    height: rect.height
-  }
+  var rect = node.getClientRects();
```

Comment:
It's a collection of DOMRect objects not a single `rect`. It might be simplified:

```ts
return node.getClientRects().map(rect => ({
  top: rect.top + documentScrollTop + scrollTop,
  left: rect.left + documentScrollLeft + scrollLeft,
  offsetWidth: node.offsetWidth,
  offsetHeight: node.offsetHeight,
  width: rect.width,
  height: rect.height
})
```